cmake/modules/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+set(SWIFTBUILD_EXPORTS_FILE ${CMAKE_CURRENT_BINARY_DIR}/SwiftBuildExports.cmake)
+
+configure_file(SwiftBuildConfig.cmake.in
```

Comment:
There's a special version of `configure_file` for generating project configs:
[`configure_package_config_file`](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html)